manager/controllers/app/blueprint_controller.go

```diff
@@ -233,6 +234,7 @@ func (r *BlueprintReconciler) reconcile(ctx context.Context, log logr.Logger, bl
 		}
 		module.Arguments.Labels[app.BlueprintNameLabel] = blueprint.Name
 		module.Arguments.Labels[app.BlueprintNamespaceLabel] = blueprint.Namespace
+		module.Arguments.Labels[app.DataAccessModuleNamespaceLabel] = dataAccessModuleNamespace
```

Comment:
I believe the module.Arguments.Labels will end up being passed to the data access module in the .Values.labels. If that is true then we cannot add this label if dataAccessModuleNamespace == default blueprints namespace, until we fully implement pass 2. For pass 1, the chart is creating the namespace. And if this is the default blueprints namespace then the helm install will fail if this label is set.

Reply:
I agree that this line should be removed, we don't need it for neither pass 1 nor pass 2.

Reply:
Removed.

Reply:
Actually we need this for pass 1. If the , if the dataAccessModuleNamespace label is specified and is != defaultModuleNamespace then set it. if the dataAccessModuleNamespace label is specified == defaultModuleNamespace ; don't set it. We need this label to be conditionally set in the values.labels so data access module can create it if NOT the default.